CMSgov/design-system
# [WNMGDS-973] - Convert Review component to typescript

**Please follow the format below and remove any sections that aren't relevant.**
## Summary
Updated Review component to use condensed functional typescript syntax

### Added
Added an ignore to our extractReactData gulp runner so it would not try to extract props from storybook stories

## How to test
run `yarn build && yarn start`, run `yarn test`, ensure no changes or new errors
